iotedgedev/modules.py

```diff
@@ -15,6 +14,53 @@ def __init__(self, envvars, utility, output, dock):
         self.dock = dock
         self.dock.init_registry()
 
+    def create(self, name, template):
+        self.output.header("CREATING MODULE")
+
+        repo = "{0}/{1}".format(self.envvars.CONTAINER_REGISTRY_SERVER, name.lower())
+        cwd = self.envvars.MODULES_PATH
+        if template == "csharp":
+            cmd = "dotnet new -i Microsoft.Azure.IoT.Edge.Module"
```

Comment:
We'll need to have a method that verifies dotnet is installed.  There's another PR open for that.

Reply:
Thanks. Then shall we leave this feature to that PR?